onnxscript/tools/benchmark/export_model_test.py

```diff
@@ -6,7 +6,11 @@
 import unittest
 
 import onnxscript.tools.benchmark.export_model
-from onnxscript.tools.transformers_models import has_transformers
+from onnxscript._internal.version_utils import (
+    has_transformers,
+    is_onnxruntime_training,
+    torch_older_than,
+)
```

Comment:
```suggestion
from onnxscript._internal import version_utils
```

nit

Reply:
The lines become longer and the linter split them over multiple line. It is easier to read that way.